composer.json

```diff
@@ -19,6 +19,7 @@
         "ezyang/htmlpurifier": "4.13.0",
         "google/apiclient": "2.12.0",
         "guzzlehttp/guzzle": "7.4.0",
+        "html2text/html2text": "^4.3",
```

Comment:
would pin this to exact version of 4.3.1

Reply:
good catch, I really did do this at one point but the pain of `docker cp composer.lock` with easy dev distracted me :)